lib/Yam/Agama/Pom/GrubMenuLeapPage.pm

```diff
@@ -25,6 +26,21 @@ sub expect_is_shown {
 }
 
 sub edit_current_entry { shift->{grub_menu_base}->edit_current_entry() }
-sub select_first_entry { shift->{grub_menu_base}->select_first_entry() }
+sub select_first_entry {
```

Comment:
functions that contains logic or that override should go on top.
Let's leave the so call `by-passing` (I don't remember the exact name) ones on the bottom of the file, like `edit_current_entry`, they look all similar, one-line thingy by-product of the composition.